esmvaltool/references/alexander.bibtex

```diff
@@ -0,0 +1,11 @@
+@article{alexander,
```

Comment:
This should be `alexander[YY]jgr` and the reference in the list should also be fixed.

Reply:
Thanks @mattiarighi , this is now fixed.